src/main/archetype/dispatcher.cloud/src/conf.dispatcher.d/filters/filters.any

```diff
@@ -8,3 +8,6 @@ $include "./default_filters.any"
 
 # Allow components JSON model
 /0101 { /type "allow" /extension "json" /selectors "model" /path "/content/*" }
+
+# Allow webmanifest extension
+/0102 { /type "allow" /extension "webmanifest" /path "/content/*" }
```

Comment:
shall we limit the rule to the file name as well? That name isn't supposed to change